linode/manager
# refactor: [M3-7537] - PaginationControls Storybook v7 Story

## Description 📝
- Migrate PaginationControls component story to a V7 (.tsx) story

## Changes  🔄
- migrate to v7 story
- update comments + tests

## How to test 🧪

### Verification steps 
- confirm story loads http://localhost:6006/?path=/docs/components-pagination-control--documentation
- confirm pagination still works on CM
- `yarn test PaginationControls`

## As an Author I have considered 🤔

*Check all that apply*
